Biotech/Patches/ThingDefs_Races/Races_Mechanoids_Medium.xml

```diff
@@ -1,15 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Patch>
 
-	<!-- ========== Cover-Seeking Behavior ========== -->
-
-	<Operation Class="PatchOperationAdd">
-		<xpath>Defs/PawnKindDef[defName="Mech_Tesseron" or defName="Mech_Apocriton"]</xpath>
```

Comment:
Given that the tesseron and apocriton are a support/command mech and a ranged fighter respectively, why wouldn't we want them seeing cover?

Reply:
The Tesseron tends to hit the cover it stays behind and the behavior conflicts with the Apocriton's default behavior of seeking dead allied mechs and resurrecting them, they also have a jump pack that let's them avoid close range combat, them going behind cover conflicts with both of those behaviors.

Reply:
Fair enough.